cmd/bucket-encryption.go

```diff
@@ -0,0 +1,228 @@
+/*
```

Comment:
This file can be moved to `pkg/bucket/encryption/`  as a proper package with its own unit tests for parsing.

Reply:
done